docs/docsite/rst/presentations/openstack-modules-intro.rst

```diff
@@ -0,0 +1,394 @@
+==================================
+Intro to Ansible OpenStack Modules
+==================================
+
+This document contains a presentation in `presentty`_ format. If you want to
+walk through it like a presentation, install `presentty` and run:
+
+.. code:: bash
+
+    presentty docs/docsite/rst/presentations/openstack-modules-intro.rst
+
+The content is hopefully helpful even if it's not being narrated, so it's being
+included in the `shade` docs.
+
+.. _presentty: https://pypi.python.org/pypi/presentty
+
+Ansible: Community, Collaboration, and the OpenStack Modules you know and love
+==============================================================================
+
+This talk is Free Software
+==========================
+
+* Written for presentty (https://pypi.python.org/pypi/presentty)
+* docs/docsite/rst/presentations/openstack-modules-intro.rst
+
+Ansible OpenStack Modules
+=========================
+
+* http://docs.ansible.com/ansible/list_of_cloud_modules.html#openstack
+* focused on consuming OpenStack APIs
+* Some are end-user focused
+* Some are deployer focused
+
+Work on All OpenStack Clouds
+============================
+
+Don't let the existence of Rackspace modules fool you. The OpenStack
+modules work just great on Rackspace.
+
+Work Around Deployer Differences ... To a Point
+===============================================
+
+* Herculean effort is made to hide a giant amount of differences
+* Even so - some bugs are EWONTFIX
+
+A provider decided to redefine the OpenStack AZ concept complete
+with API changes. That is unsupportable.
+
+Based on Shade
+==============
+
+* abstracts deployment differences
+* designed for multi-cloud
+* simple to use
+* massive scale
+
+  * optional advanced features to handle 20k servers a day
+
```

Comment:
The single indented bullet point isn't necessary if it is the only point. You may want to promote the single bullet point up to the bullet it belongs to but put it in parenthesis.